### DISCUSSION TITLE
How to modify the mail template?

### DISCUSSION THREAD

**Discussion by Dofine-dufei** [kubesphere/kubesphere#3381] [General]
After I enable notification model,  it works, and I get an alarm mail,but I don't need the title 'Kubeshere Notification' ...
I need modify my templete, and how to do?

**Comment by FeynmanZhou** [comment]
@wanjunlei As I remember that the mail template is not allowed to modified in KubeSphere v3.0.0, right?

**Reply by wanjunlei** [reply]
Yes
